### PR TITLE
updated from Dev 492

### PR DESCRIPTION
**Pull request by ErezPasternak**

### CODE REVIEW

**Comment by ErezPasternak** [comment]
@BenyH please review

File: Setup/shield-version.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
-#Build Staging:Build_485 on 19/02/13   #UNR#
-SHIELD_VER=8.0.0.latest SHIELD_VER=Staging:Build_485
+#Build Staging:Build_492 on 19/02/27   #UNR#
```
Comment (ozlevka-ericom) [inline Setup/shield-version.txt]: Consider removing #UNR#...
The update will be more simple

Comment (BenyH) [reply Setup/shield-version.txt]: I agree, I dont see a reason to restart

**Comment by ErezPasternak** [comment]
@BenyH do we need to update other files ?

**Pull request merged**